addons/common/init_perFrameHandler.sqf

```diff
@@ -155,10 +161,13 @@ if (isMultiplayer) then {
     [QFUNC(missionTimePFH), {
         SCRIPT(missionTimePFH_sp);
         if (time != GVAR(lastTime)) then {
-            CBA_missionTime = CBA_missionTime + (_tickTime - GVAR(lastTickTime)) * accTime;
+            private _additionCheckPrecision = CBA_missionTime + (_tickTime - GVAR(lastTickTime)) * accTime;
+            if (CBA_missionTime == _additionCheckPrecision) exitWith {}; // used to detect floating point precision error
+            CBA_missionTime = _additionCheckPrecision;
             GVAR(lastTime) = time; // used to detect paused game
+            GVAR(lastTickTime) = _tickTime;
+        } else {
+            GVAR(lastTickTime) = _tickTime;
         };
-
-        GVAR(lastTickTime) = _tickTime;
```

Comment:
???

Reply:
We don't want to update `GVAR(lastTickTime) = _tickTime;` if `CBA_missionTime == _additionCheckPrecision` return true